ledger-service/http-json-perf/src/main/scala/com/daml/http/perf/scenario/SyncQueryVariableAcs.scala

```diff
@@ -43,7 +43,8 @@ class SyncQueryVariableAcs
       }
 
   setUp(
-    fillAcsScenario(wantedAcsSize, silent = true).inject(atOnceUsers(1)),
-    syncQueryScenario.inject(atOnceUsers(1)),
+    fillAcsScenario(wantedAcsSize, silent = true).inject(atOnceUsers(defaultNumUsers)).andThen {
+      syncQueryScenario.inject(atOnceUsers(1))
```

Comment:
query scenario back to 1 since running with multiple users was not working, given 500 runs this is not a significant slowdown.